pkgs/by-name/on/oncall/package.nix

```diff
@@ -55,9 +28,17 @@ python.pkgs.buildPythonApplication rec {
 
     # Log Python errors to uwsgi
     ./verbose_logging.patch
+
+    # Add support for Falcon 4
+    # https://github.com/linkedin/oncall/pull/433
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/linkedin/oncall/pull/433.patch";
+      hash = "sha256-JiDzN3cb8LHZun95mL8HUj6xMrGPish2xhG+zE/1xVA=";
+      name = "falcon_4_support.patch";
+    })
```

Comment:
Please don't use fetchpatch for resources that might change (yours will change the moment you add commits to the pr or rebase)
see [README.md#patches](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#patches)

Reply:
You didn't really fix the issue, the patch url will still be garbage collected when for what ever reason you rebase your changes in the pr. I went ahead and also added a branch to track that commit so it should be somewhat safe now. But Please keep this in mind for future changes to nixpkgs. Such changes should be generally vendored according to [README.md#patches](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#patches). But i won't insist on it due to the patches insane size.